subgraph/src/datasources/FortaStaking.ts

```diff
@@ -117,24 +166,36 @@ function updateStake(
     _subject,
     _staker
   );
+
+  const updatedStakeTotalShares: BigInt = stake.shares ? stake.shares as BigInt : BigInt.fromI32(0);
+  const updatedStateInActiveShares: BigInt = stake.inactiveShares ? stake.inactiveShares as BigInt: BigInt.fromI32(0);
+
+  updateAggregateStake(_stakerId, prevStakeTotalShares, prevStateInActiveShares, updatedStakeTotalShares, updatedStateInActiveShares)
```

Comment:
so, we should use it in the way for scanner pools, correct?

Reply:
Yep I think we could reuse the same function.